text/0000-router-service.md

```diff
@@ -400,6 +437,3 @@ Should we introduce new API via the `Ember` global and switch to a module export
 
 They let you do spooky-action-at-a-distance stuff. We could instad choose to implement single-purpose methods like `(get-route-info)` and `{{#with-route-info someValue}}` that would solve the routing case without opening the door to arbitrary user-defined ones.
 
-## Possible semantic breakage of willTransition
```

Comment:
Agreed that this is far too large of a breaking change. The reason we pushed to get #126 onto the agenda was because we reached a point in which we can't continue to maintain all of our "`willTransition` fires every time" hacks any longer. I know of at least four addons (three of which are internal) break along this change.